packages/venia-concept/src/components/ProductFullDetail/ProductFullDetail.js

```diff
@@ -0,0 +1,112 @@
+import { Component, createElement } from 'react';
+import {
+    bool,
+    func,
+    shape,
+    number,
+    objectOf,
+    arrayOf,
+    string
+} from 'prop-types';
+import { Price } from '@magento/peregrine';
+
+import classify from 'src/classify';
+import Carousel from 'src/components/ProductImageCarousel';
+import Quantity from 'src/components/ProductQuantity';
+import RichText from 'src/components/RichText';
+import defaultClasses from './productFullDetail.css';
+
+class ProductFullDetail extends Component {
+    static propTypes = {
+        classes: objectOf(string).isRequired,
+        product: shape({
+            id: number,
+            sku: string.isRequired,
+            price: shape({
+                regularPrice: shape({
+                    amount: shape({
+                        currency: string.isRequired,
+                        value: number.isRequired
+                    })
+                }).isRequired
+            }).isRequired,
+            media_gallery_entries: arrayOf(
+                shape({
+                    label: string,
+                    position: number,
+                    disabled: bool,
+                    file: string.isRequired
+                })
+            ),
+            description: string
+        }).isRequired,
+        onClickAddToCart: func.isRequired
```

Comment:
I think `addToCart` would be a better name. Not crazy to think this could be used in the component for other interactions that trigger add to cart. Touch/enter/etc

Reply:
Agreed.